source/renderer/app/api/ada/changeAdaWalletPassphrase.js

```diff
@@ -1,25 +1,25 @@
 // @flow
-import type { AdaWallet } from './types';
-import { request } from './lib/request';
+import type { AdaV1Wallet } from './types';
+import { request } from './lib/v1/request';
 import { encryptPassphrase } from './lib/encryptPassphrase';
 
 export type ChangeAdaWalletPassphraseParams = {
   ca: string,
   walletId: string,
-  oldPassword: ?string,
-  newPassword: ?string,
+  oldPassword?: string,
```

Comment:
@daniloprates 
Yeah, it can be just completely omitted from the request body like this. As I understand the syntax here: `oldPassword?: string` means the property `oldPassword` can be left off the object completely without raising an error. `oldPassword: ?string` means that the property `oldPassword` will always be on the object, but it may or may not be a string. It could be null, it could be undefined, it could be an empty string etc. Were you getting any particular errors here?

Reply:
No errors, I just was just wondering if the "?" was necessary. But if it can be omitted, it's fine.